www/templates/diary/infinite_scroll_list.html

```diff
@@ -5,7 +5,7 @@
         <button class="button button-icon" ng-click="increaseHeight()">
             <i class="icon ion-arrow-resize"></i>
         </button>
-        <div class="row buttons labelfilterlist">
+        <div class="buttons labelfilterlist">
```

Comment:
why does this fix the issue "buttons collapsing" issue? Why did it happen in the first place?

Reply:
I don't have a specific answer, but I do know that "row" applied a `width: 100%` to the `<div>`, which tends to cause problems with flexbox layouts.
The parent is a flexbox `<span>` and it is not big enough to fit all the buttons anyway, so they overflow. There's a lot of weird stuff happening in the CSS here.

Reply:
Thanks for letting me know. Good thing we are going to rewrite in a non-classic CSS framework...